elixir/apps/web/test/support/acceptance_case.ex

```diff
@@ -134,7 +134,8 @@ defmodule Web.AcceptanceCase do
     if env = System.get_env("E2E_DEFAULT_WAIT_SECONDS") do
       String.to_integer(env)
     else
-      2
+      # GitHub shared runners can be very slow during peak hours
```

Comment:
https://github.com/firezone/firezone/actions/runs/8725396789/job/23938263443

Reply:
@jamilbk it's already 5 seconds on CI: [ E2E_DEFAULT_WAIT_SECONDS in ](https://github.com/firezone/firezone/blob/0c7c96dd074d1bde1b1c8f3855503365a154768a/.github/workflows/_elixir.yml#L279)

Reply:
😢